backend/src/seeder/course-seeder/course-seeder.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { Course } from '../../course/course.entity';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+
+@Injectable()
+export class CourseSeederService {
+  constructor(
+    @InjectRepository(Course)
+    private readonly courseRepository: Repository<Course>,
+  ) {}
+
+  async seedCourse() {
+    // placeholder
+    const course = new Course();
+    course.year = 111;
+    course.semester = 0;
+    course.serialNo = 1001;
+    course.classNo = 'SE6030*';
+    course.title = '軟體工程實務';
+    course.credit = 1;
+    course.passwordCard = 'passwordCard';
+    course.teachers = ['["梁德容", "王尉任", "鄭永斌"]'];
+    course.classTimes = ['["1-2", "1-3", "1-4"]'];
```

Comment:
from
```
course.teachers = ['["梁德容", "王尉任", "鄭永斌"]'];
course.classTimes = ['["1-2", "1-3", "1-4"]'];
```
to
```
course.teachers = '["梁德容", "王尉任", "鄭永斌"]';
course.classTimes = '["1-2", "1-3", "1-4"]';
```

Reply:
But the data type of course.teachers and course.classTimes are string[]

Reply:
The export type of `Course` is wrong. Just correct it to `string`. Thanks!